carsus/io/cmfgen/base.py

```diff
@@ -292,7 +294,7 @@ def load(self, fname):
         kwargs = {}
         kwargs['header'] = None
         kwargs['index_col'] = False
-        kwargs['sep'] = '\s*-?\s+-?|(?<=[^edED])-|(?<=Pe)-'  # TODO: this regex needs some review
+        kwargs['sep'] = '\s*-?\s+-?|(?<=[^edED])-|(?<=[FDP]e)-'  # TODO: this regex needs some review
```

Comment:
who should review this regex?

Reply:
At the beggining I thought about asking Alice, because I wasn't so confident about my regexes. Now I think I just have to remove that comment.